### ISSUE TITLE
5V vs 3.3V

### ISSUE THREAD

**Issue by flexiblestoic** [luisomoreau/arduinoUno-wisol#2]
I think there is a a problem in using the Wisol BRKWS01 with the Arduino Uno because one is 5V and the other 3.3V. Probably need a voltage step down logic converter. 

If you are using a 3.3V Arduino or a 3.3V card like the ESP8366, there shouldn't be a problem.

**Comment by luisomoreau** [comment]
On the Arduino I have, I have a 3.3V output (on the picture on the readme, it is just right next to the 5v output).

**Comment by flexiblestoic** [comment]
I also have the same card (Arduino Uno). The issue isn't the 3.3v power output which is fine to power the Wisol, but the level on the RX/TX pins. When I first did my tests on the Arduino Uno, I didn't really think of it but measured 5v on Wisol's data pins which is outside of its specs (I was afraid I burned it but it fortunately surivived)

It is much safer to use a 3.3v card or a tension divider or step down logic converter if 5V Arduino is used.

Some discussion on the matter: 
https://forum.arduino.cc/index.php?topic=10391.0
